src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -70,6 +70,23 @@ public static boolean isNonZeroUnsignedInteger(String s) {
         }
     }
 
+    /**
+     * Returns Integer if {@code s} represents an unsigned integer
+     * e.g. 0, 1, 2, 3, ..., {@code Integer.MAX_VALUE} <br>
+     * Will return null for any other non-null string input
+     * e.g. empty string, "-1", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
+     * @throws NullPointerException if {@code s} is null.
+     */
```

Comment:
Hi, do leave an empty line between the description and the `@throws` as part of our standardisation of Javadocs format. 😄